project/Settings.scala

```diff
@@ -40,7 +40,7 @@ object Settings {
 
   // recommended scalac options by https://tpolecat.github.io/2017/04/25/scalac-flags.html
   lazy val commonCompilerSettings = Seq(
-    "-target:jvm-1.8",
+    "-target:jvm-11",
```

Comment:
Also on L24 you can do `javacOptions ++= Seq("--release", "11"),`

(I think `--release` replaces `-source` and `-target` in Java 11.)